openshift/template.yaml

```diff
@@ -394,39 +401,42 @@ objects:
   - apiVersion: batch/v1
     kind: Job
     metadata:
-      name: superuser-job
+      name: superuser-job-1
     spec:
       parallelism: 1
       completions: 1
       activeDeadlineSeconds: 1800
       backoffLimit: 3
       template:
         metadata:
-          name: superb-job
+          name: superuser-job-1
         spec:
           containers:
-            - name: superb-job
+            - name: superuser-job-co
               image: "${REGISTRY_IMAGE}:${IMAGE_TAG}"
               command:
                 ["python3", "./manage.py", "createsuperuser", "--noinput"]
+              startupProbe:
```

Comment:
Same comment as above.

Reply:
We really really don't need any Probes in Job (Job is not a long-running Process). It's just a placeholder.